assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -18,13 +23,59 @@ import EmptyCart from './empty-cart';
 /**
  * Component to handle edit mode of "Cart Block".
  */
-const CartEditor = ( { className } ) => {
+const CartEditor = ( { className, attributes, setAttributes } ) => {
+	const { isShippingCalculatorEnabled, isShippingCostHidden } = attributes;
+
 	const [ isFullCartMode, setFullCartMode ] = useState( true );
 
 	const toggleFullCartMode = () => {
 		setFullCartMode( ! isFullCartMode );
 	};
 
+	const getBlockSettings = () => (
+		<InspectorControls>
+			<PanelBody
+				title={ __(
+					'Shipping calculations',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<ToggleControl
+					label={ __(
+						'Shipping calculator',
+						'woo-gutenberg-products-block'
+					) }
+					help={ __(
+						'Allow customers to estimate shipping',
```

Comment:
I think `help` texts should end with a period.

Reply:
Agreed - the core blocks all use sentence case and full stop for the help text, we should be consistent.

Reply:
Also, I find this one a little confusing, are the customers "estimating" shipping or selecting a shipping option (newbie question)?

Reply:
estimating shipping should allow users to enter an address and see what the cost for that place is with each shipping method